src/character_attire.cpp

```diff
@@ -1945,6 +1950,103 @@ void outfit::absorb_damage( Character &guy, damage_unit &elem, bodypart_id bp,
     }
 }
 
+std::string outfit::get_liquid_descriptor( int liquid_remaining )
+{
+    std::string liquid_descriptor;
+    std::string liquid_descriptor_string;
+    if( liquid_remaining <= 10 ) {
+        liquid_descriptor = "droplets of";
+    } else if( liquid_remaining <= 20 ) {
+        liquid_descriptor = "a glob of";
+    } else if( liquid_remaining <= 30 ) {
+        liquid_descriptor = "a spurt of";
+    } else if( liquid_remaining <= 50 ) {
+        liquid_descriptor = "a splatter of";
+    } else if( liquid_remaining <= 75 ) {
+        liquid_descriptor = "a spray of";
+    } else if( liquid_remaining <= 100 ) {
+        liquid_descriptor = "quite a lot of";
+    } else if( liquid_remaining <= 125 ) {
+        liquid_descriptor = "copious amounts of";
+    } else if( liquid_remaining <= 150 ) {
+        liquid_descriptor = "a cascade of";
+    } else if( liquid_remaining <= 175 ) {
+        liquid_descriptor = "a torrent of";
+    } else if( liquid_remaining <= 200 ) {
+        liquid_descriptor = "a flood of";
+    } else {
+        liquid_descriptor = "a great deluge of";
+    }
+    liquid_descriptor_string += string_format( _( "%s" ), liquid_descriptor );
```

Comment:
Sorry, I think I mislead you in the other PR. The function as written will not pass the strings to translation, it would pass the string `%s` for translation, which... well, that's not very helpful, is it?

Each individual English string that's hardcoded basically needs to be prepended/appended as `_("my_string")`. It should be fine to just wrap each of the strings you've added (e.g. `_("a spurt of")` and ``_("droplets of")`` so on so forth).

Reply:
Oh OK, thanks again for the help 😄